### DISCUSSION TITLE
Donut upgrade

### DISCUSSION THREAD

**Discussion by clcoffey** [InnerSourceCommons/InnerSourceMarketing#134] [General]
Over the past weeks, we have missed the Donut introductions in the InnerSource Commons slack community.

We currently have 88 members in the Donut channel and with the free tier, the donut will only make 12 introductions each time. Our team got in touch with the Donut team and they offered a 50% cut in their prices for the paid tier.

Some options we currently have:

1. To upgrade the donut for all the 88 members of the channel. 
         - This will cost: 
              - $100/month billed annually. That is $1200/year (including the discount)
              - $120/month billed monthly. That is $1440/year (including the discount)
2. To create a new channel and ask people to rejoin. 
         - Considering that there are many people that are not engaging in conversations in the current channel, we could create a new channel and ask people that are interested to join. 
         - This will ensure that the members of the new channel are actively interested in connecting and meeting other people in the community.
         - Also, if the number of people in the new channel is under 50, we will pay:
                - $50/month billed annually. That is $600/year (including the discount)
                - $60/month billed monthly. That is $720/year (including the discount)
3. The third alternative is building our own Donut app. 
        - I found this blog post: https://medium.com/xandr-tech/bagel-bot-reimaging-donut-1fa68154a80c
        - I followed the instructions and tested the code in a separate Slack account and with small changes I managed to make it work. 
        - For this, we will need help in creating a user interface for the app and finding solutions for hosting it?
4. We are open to any other suggestions.

**Comment by mishari** [comment]
There's a list of alternatives here as well https://alternativeto.net/software/donut-for-slack/

**Reply by clcoffey** [reply]
Thanks @mishari. 2 of the 3 alternatives on that website seem viable:
1. [Snack](https://aboutsnack.com/) - which claim to be free for online communities and conferences. 
2. [Catchup](https://catchup.rocks/) - which costs $29/month for up to 100 users matched. 

I can get in contact with them and find out further information

**Reply by clcoffey** [reply]
Looking more into it Snack is about creating spontaneous short video meetings. https://aboutsnack.com/video-chat-roulette I am not sure if that fits with what we need.

**Comment by rrrutledge** [comment]
What is the pricing model for donut?  Pay per channel user?

**Reply by clcoffey** [reply]
Yes. 

For 25-49 people in the channel they charge:
- $50/month billed annually. That is $600/year (including the discount)
- $60/month billed monthly. That is $720/year (including the discount)

And the next tier is 50-99 people in the channel with the following charges:
- $100/month billed annually. That is $1200/year (including the discount)
- $120/month billed monthly. That is $1440/year (including the discount)

**Comment by claredillon** [comment]
So, I there are a couple of things I am thinking of/assumptions I am making:

1. We want to minimize the number of services we are managing / maintaining ourselves as an org
2. We have to manage costs, and consider costs if the services scale / become popular (FYI - we're running at just less than 200 active members per month in Slack as people come and go - and that spikes around the Summits)
2. We want to maximise the number of Donut-type engagements (because they are so valuable to our community)
3. We want to minimize the change/overhead for folks who have signed up, noting:
  - it's totally valid to want to Donut, but not to have been able to for the last few goes 
  - the overhead of keeping the numbers in the channel low for cost reasons (e.g. creating new channels etc.) is not insignificant over time.  

So taking all that into account, and given that In general I don't like the idea of introducing manual processes (in particular those that can inconvenience community folk) just to manage costs. 

Therefore - I would propose we make Donut operational again, committing only on a month by month basis for up to 100 in the channel  and look at other alternatives, which may be cheaper to scale as we might like longer term...

**Reply by clcoffey** [reply]
I have emailed Catchup about a price for Non-profits, I will let you know when they come back to me.

**Comment by clcoffey** [comment]
Reply from Catchup:

Dear Cristina,

thank you for your email. We offer a 50% discount for non-profits, i.e.
- Free (match up to 30 users per channel, 1 coffee channel max)
- Builder for $14.50/month instead of $29/month (match up to 100 users per channel, unlimited number of coffee channels)
- Enterprise for $49.50/month instead of $99/month (match unlimited users per channel, unlimited number of coffee channels)

We also offer a 1-month free Enterprise trial during which you can test the complete functionality of the Enterprise plan for free (unlimited users matched / unlimited number of coffee channels). No credit card required - please just let me know via email when you would like to start your trial.

Please let me know in case you have any other questions / would like to start the free 1 month trial. You can also find more information in our FAQ https://catchup.rocks/faq or book a call via our Calendly

Best,

Stefan

**Comment by clcoffey** [comment]
I propose to give Catchup a try for 1 month starting Monday with introductions every 2 weeks. During this time we will disable the Donut app and test only Catchup. Catchup is advertising the same services as the Donut so we shouldn't feel the difference. But we won't know until we try.

If anyone is opposed to this please let me know within the next 72 hours. If there are no objections or comments I will go ahead with the test starting Monday.

I would also appreciate your feedback on the frequency of the introductions. Is 2 weeks a good timeframe in between meetings or should we move back to every week?

**Reply by claredillon** [reply]
Every 2 weeks works for me.

**Comment by mishari** [comment]
My vote is for a weekly meet.

**Comment by claredillon** [comment]
Just recording that as of 10:52 am UTC 24/01/22 the poll in Slack resulted in 4 votes for weekly and 12 votes for every two weeks. 

Plus noting I got my first Catch-up proposal - all pretty seamless / very familiar process - so thanks @clcoffey for this!!

**Comment by clcoffey** [comment]
Thanks @claredillon, also Isabel just confirmed that she removed the Donut from the channel so we will not have any further updates from it unless we decide to move back after this trial.

**Comment by spier** [comment]
Catch-up seems to be working fine?

@claredillon @clcoffey shall we close this discussion?

**Reply by clcoffey** [reply]
sounds good. I think this was fixed.
